cephci/stats.py

```diff
@@ -0,0 +1,122 @@
+from datetime import datetime, timezone
+
+from docopt import docopt
+from utils.configs import get_cloud_credentials, get_configs, get_reporting_credentials
+from utils.utility import set_logging_env
+
+from cli.cloudproviders import CloudProvider
+from cli.exceptions import ConfigError
+from cli.reports.database import Database
+from utility.log import Log
+
+# Set logging env
+LOG = Log(__name__)
```

Comment:
Add separate comment for global variable

Reply:
Done